includes/api/class-edd-api-v2.php

```diff
@@ -431,7 +431,7 @@ public function get_recent_sales() {
 					$c++;
 				}
 
-				$sales['sales'][ $i ]['discounts'] = $discount_values;
+				$sales['sales'][ $i ]['discounts'] = ( ! empty( $discount_values ) ? $discount_values : null );;
```

Comment:
Whoops not sure how that didn't get picked up as an error when I tested before pushing and been using the same code since...